packages/pagination/stories.js

```diff
@@ -51,54 +51,56 @@ storiesOf('Pagination Container', module)
       });
 
       return (
-        <ul {...getContainerProps({ style: { display: 'flex', role: 'navigation' } })}>
-          <li
-            {...getPreviousPageProps({
-              current: selectedItem === 'prev',
-              focused: focusedItem === 'prev',
-              item: 'prev',
-              focusRef: previousPageRef,
-              ref: previousPageRef,
-              key: 'previous-page'
-            })}
-          >
-            Prev
-          </li>
-          {pages.map((page, index) => {
-            return (
-              <li
-                {...getPageProps({
-                  focused: index === focusedItem,
-                  current: index === selectedItem,
-                  page: index,
-                  item: index,
-                  focusRef: pageRefs[index],
-                  ref: pageRefs[index],
-                  key: `page-${index}`,
-                  style: {
-                    outline: index === focusedItem && '3px solid red',
-                    background: index === selectedItem && 'gray',
-                    padding: '0 6px'
-                  }
-                })}
-              >
-                {index + 1}
-              </li>
-            );
-          })}
-          <li
-            {...getNextPageProps({
-              current: selectedItem === 'next',
-              focused: focusedItem === 'next',
-              item: 'next',
-              focusRef: nextPageRef,
-              ref: nextPageRef,
-              key: 'next-page'
+        <nav {...getContainerProps({ role: null })}>
```

Comment:
Why is `[role="null"]` being applied?

Reply:
Because the role is inherent in the semantics of `<nav>`. We expose all `role` (and `ariaLevel`, for example) attributes for this kind of override where the consumer can express they know exactly what they are doing.

Reply:
Could we leave a comment for this? We might see some people copy/pasting this example and leaving `role` the way it's shown.

Reply:
Added comment